src/librustdoc/html/format.rs

```diff
@@ -961,7 +961,7 @@ fn fmt_type<'cx>(
                 write!(f, "impl {}", print_generic_bounds(bounds, cx))
             }
         }
-        clean::QPath { ref name, ref self_type, ref trait_, ref self_def_id } => {
+        clean::QPath { assoc: ref last, ref self_type, ref trait_, ref self_def_id } => {
```

Comment:
Another `last` remnant.

Reply:
agh, thanks. I seem to be quite forgetful when it comes to refactors.